watcher.js

```diff
@@ -0,0 +1,133 @@
+/*
+ * backpack-node-sass
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const ora = require('ora');
+const sass = require('node-sass');
+const chokidar = require('chokidar');
+const importer = require('node-sass-tilde-importer');
+const functions = require('bpk-mixins/sass-functions.js');
+
+const getCssFileName = name => name.replace(/\.scss/, '.css');
+
+const compileSass = file =>
```

Comment:
How about Promisify for this? it'll be much neater

Reply:
Done! let me know what you think.